repositories/tasks/deleter_repository.go

```diff
@@ -0,0 +1,8 @@
+package tasks // import github.com/mlimaloureiro/golog/repositories/tasks
+
+// DeleterTaskRepositoryInterface is a TaskRepositoryInterface that can also delete tasks directly
+type DeleterTaskRepositoryInterface interface {
```

Comment:
what was the reason to have a dedicated interface?

Reply:
Some repositories might have implementations for this 2 operations (for example a SQL repository). This interface would allow TaskService to call those methods directly instead of using GetTasks() -> SetTasks() to delete tasks. Something similar to StarterPauserTaskRepositoryInterface.

But currently no repositories implement it.

Should i remove it?